### PR TITLE
Vatrp-2127:iOS: sip Spam prevention

### PR DESCRIPTION
**Pull request by karen-accontech**
Added functionality to prevent sip spam.

### CODE REVIEW

File: Classes/LinphoneCoreSettingsStore.m [hunk 1]
```diff
@@ -283,7 +283,8 @@ - (void)transformLinphoneCoreToKeys {
         }
 		[self setCString:preset ? preset : "high-fps" forKey:@"video_preset_preference"];
         MSVideoSize vsize;
-        
+        vsize.width = 0;
+        vsize.height = 0;
```
Comment (zackm0571) [inline Classes/LinphoneCoreSettingsStore.m:287]: What is this doing?

File: Classes/LinphoneCoreSettingsStore.m [hunk 2]
```diff
@@ -768,6 +760,8 @@ - (BOOL)synchronize {
         }
 		linphone_core_set_video_preset(lc, [videoPreset UTF8String]);
 		MSVideoSize vsize;
+        vsize.width = 0;
+        vsize.height = 0;
         NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```
Comment (zackm0571) [inline Classes/LinphoneCoreSettingsStore.m:765]: What is this doing?

File: Classes/LinphoneCoreSettingsStore.m (hunk 1 above)
Comment (karen-accontech) [inline Classes/LinphoneCoreSettingsStore.m:287]: Just the initial initialisation of variable

Xcode 7.3 gives error without this initialization. 
"Variable 'vsize' is used uninitialized whenever 'if' condition is false"

File: Classes/LinphoneCoreSettingsStore.m (hunk 2 above)
Comment (karen-accontech) [inline Classes/LinphoneCoreSettingsStore.m:765]: Just the initial initialisation of variable

Xcode 7.3 gives error without this initialization. 
"Variable 'vsize' is used uninitialized whenever 'if' condition is false"

**Comment by zackm0571** [comment]
+1

**Pull request merged**